index.js

```diff
@@ -1,38 +1,255 @@
+//Each object has to store: name, remaining boats, and their respective board.
+
+//*Each board needs to be initialized randomly with '0' and four '1' wich means the state of the cell. 
+//*Numbers 1 are representing the 4 positions of the player's ships
+
+//*Also we want you to display the name of the turn player in the tag that has the id 'turn_player'. 
+
+//And if there is a winner  a text with: 'Congratulationes {name_player}!! you win'
+
+//*in the index.html file you are going to find 4 more ids: 'name_player1' , 'name_player2' , 'ships_player1' , 'ships_player2'. 
+//*We want to see the information of each player in the respective elements
+
 //As our previous Battleship, the winner is the player that hits the 4 opponent's ships first
-//one more Thing create a 'reset' and a 'new game' buttons as childs of the element with the id 'buttons'. the reset button has to start the game again and the new game create a new game with new players and a new random board.
 
+//*one more Thing create a 'reset' and a 'new game' buttons as childs of the element with the id 'buttons'. 
+//*the reset button has to start the game again and the new game create a new game with new players and a new random board.
+
+// Create Players
+var shipNum = "4";
+
+let playerOne = {
+  name: prompt("playerOne - What is your first name?"),
+  shipCount: parseInt(shipNum),
+  gameBoard: [
+    [0,0,0,0],
+    [0,0,0,0],
+    [0,0,0,0],
+    [0,0,0,0]
+  ]
+}
+
+let playerTwo = {
+  name: prompt("playerTwo - What is your first name?"),
+  shipCount: parseInt(shipNum),
+  gameBoard: [
+    [0,0,0,0],
+    [0,0,0,0],
+    [0,0,0,0],
+    [0,0,0,0]
+  ]
+}
+
+// Define name for player, boardPlayer and ships 
+const namePlayer1 = document.getElementById('name_player1');
+const namePlayer2 = document.getElementById('name_player2');
+const lives1 = document.getElementById('ships_player1');
+const lives2 = document.getElementById('ships_player2');
 const board_Player1 = document.getElementById('board_player1');
+const board_Player2 = document.getElementById('board_player2');
+namePlayer1.textContent = playerOne.name;
+namePlayer2.textContent = playerTwo.name;
+lives1.innerHTML = parseInt(shipNum);
+lives2.innerHTML = parseInt(shipNum);
+
+// Create buttons 
+const buttons = document.getElementById('buttons');
+const resertButton = document.createElement('button');
+const newGameButton = document.createElement('button');
+resertButton.innerHTML = 'Resert';
+newGameButton.innerHTML = 'New game';
+
+//add new element to their parents
+buttons.appendChild(resertButton);
+buttons.appendChild(newGameButton);
+
+// Create turn player 
+const turnPlayer = document.getElementById('turn_player');
+
+// Function adding ships on the board (console)
+
+const addShips = (player) => {
+  for (var i = 0; i < shipNum; i++) {
+    let xCoordinate = Math.floor( Math.random() * (3 - 0 + 1) + 0);
+    let yCoordinate = Math.floor( Math.random() * (3 - 0 + 1) + 0);
```

Comment:
we can replace `(3 - 0 + 1) + 0` directly with `4`

Reply:
Thank you